### PR TITLE
fix(datetimepicker): add date picker parameters support

### LINKED ISSUE #2861: [datetimepicker]Support for date-picker input parameters
### What package is this for?

- [ ] React
- [x] Angular

### Describe the bug

The date-time-picker is using the carbon-date-picker base which has more input parameters, 
need support for missing parameters to specify the customization at carbon-date-picker level. 

image

### To Reproduce

> Please create a reduced test case in CodeSandbox:
> https://codesandbox.io/s/github/IBM/carbon-addons-iot-react/tree/master/examples/codesandbox

### Environment/versions:

- OS: [e.g. MacOS, Windows]
- Browser: [e.g. chrome, safari]
- `carbon-addons-iot-react` version: [e.g. `v2.60.0`]

### Specific timeline issues / requests

Do you want this work within a specific time period? Is it related to an
upcoming release?

_NB: The core contributors will try to work with your timeline, but it's not
guaranteed. The earlier you make a request in advance of a desired delivery
date, the better!_

### PR DESCRIPTION
**Pull request by sophiiae**
Closes #2861 

**Summary**

- summary_here

**Change List (commits, features, bugs, etc)**

- items_here

**Acceptance Test (how to verify the PR)**

- tests here

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here

**Things to look for during review**

- [ ] Make sure all references to `iot` or `bx` class prefix is using the prefix variable
- [ ] (React) All major areas have a `data-testid` attribute. New test ids should have test written to ensure they are not changed or removed.
- [ ] UI should be checked in RTL mode to ensure the proper handling of layout and text.
- [ ] All strings should be translatable.
- [ ] The code should pass a11y scans (The storybook a11y knob should show no violations). New components should have a11y test written.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] All components should be passing visual regression test. For new styles or components either a visual regression test should be written for all permutations or the base image updated.
- [ ] Changes or new components should either write new or update existing documentation.
- [ ] PR should link and close out an existing issue

### CODE REVIEW

**Comment by davidicus** [comment]
@sophiiae can we get these merge conflicts addressed so we can merge this PR?

File: packages/angular/src/date-time-picker/date-time-picker.component.ts [hunk 1]
```diff
@@ -244,7 +259,7 @@ export class DateTimePickerComponent implements OnChanges, OnInit {
   @Input() selected: DateTimeSelection = null;
   @Input() hasRelative = true;
   @Input() hasAbsolute = true;
-  @Input() theme: 'light' | null = null;
+  @Input() theme: 'light' | 'dark' = null;
```
Comment (zvonimirfras) [inline packages/angular/src/date-time-picker/date-time-picker.component.ts:262]:
```suggestion
  @Input() theme: 'light' | null = null;
```

**Review by zvonimirfras — APPROVED** [review]

**Comment by davidicus** [comment]
Hey @zvonimirfras can we get this updated so we can get this merged in or close out this issue? Let me know when you have pushed and I will get it merged ASAP. Thanks.

**Comment by davidicus** [comment]
I am closing for lack of response. Feel free to re-open when ready.

**Pull request closed without merging**